### PR TITLE
feat(ui): new info/setup screens

### PR DESCRIPTION
**Pull request by shanimal08**
Description
---

- moved the splashscreen to its own tauri window - preferred method from the tauri docs
- **updated file and `App.tsx` structure:** (imports are the majority of the file changes sry!)
  * moved `App.tsx` and `AppWrapper` into a dedicated new `App` dir, added a styles file and content wrapper component (for animation in and out the different sections detailed next)
  * organised the `containers` directory into three main "containers": `floating`, `phase`, and `main`
    - `main` contains the main app section content: dashboard, sidebar, airdrop stuff, etc.
    - `floating` contains all our dialogs and modals i.e "floating" elements all wrapped up in a `FloatingTree` 
    - `phase` is for stuff that phases in or out, right now that is just the `ShuttingDownScreen` and this new `Setup` flow 
  * simplified `App.tsx` way down and only include the main wrapper components for the app and each child section
- replaced lottie dependency (to one without a vulnerability)
- removed old setup UI and dupe code as i went along
- **_NEW Info Screens!_** - the juicy new stuff all starts in `src/containers/phase/Setup?*` entry here in [Setup.tsx](https://github.com/tari-project/universe/pull/969/files#diff-19e7c635e228252836066e4a64154cb03931db3ba8dd2464cc6742d808149ae6):
  * new info screen elements including: 
     - hero text
     - the info nav slider (time based but you can also click through to the next "slide" if you're impatient
     - the setup progress and analytics toggle in the footer
  * cleaned up the UI store

Motivation and Context
---

How Has This Been Tested?
---
- locally: 

https://github.com/user-attachments/assets/af38e3f8-5354-472c-9daa-561277960b9e

https://github.com/user-attachments/assets/196300d7-0e5e-472e-ad31-307538b547d0

**(UPDATE WITH LOTTIE SOON):**

https://github.com/user-attachments/assets/850116c5-c45f-4492-acb3-f939bbc6039a

https://github.com/user-attachments/assets/e110d743-d466-4e18-bb86-0f995250cca9

What process can a PR reviewer use to test or verify this change?
---

- run app 
- click around on the new info screens
- after setup make sure nothing's broken

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: src/containers/phase/Setup/components/InfoNav/InfoNav.styles.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+export const Heading = styled.div`
+    font-weight: 700;
+    font-size: calc(2.6rem + 1vmin);
+    color: ${({ theme }) => theme.palette.text.primary};
+    text-transform: uppercase;
+    line-height: 1;
+    letter-spacing: -2px;
+    white-space: pre-line;
+    text-wrap: nowrap;
```
Comment (peps) [inline src/containers/phase/Setup/components/InfoNav/InfoNav.styles.ts:20]: `text-wrap` isn't a thing (though it should be!)

File: src/containers/phase/Setup/components/SetupProgress.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+const Percentage = styled(Typography)`
+    font-weight: 700;
+    line-height: 1;
+`;
+
+const InfoText = styled(Typography)`
+    font-weight: 400;
+    margin-bottom: 14px;
+    text-wrap: nowrap;
```
Comment (peps) [inline src/containers/phase/Setup/components/SetupProgress.tsx:21]: `text-wrap` wont do anything `white-space: nowrap;`

**Review by peps — COMMENTED** [review]
Overall it looks great, I just have some nit picky feedback. 

- I think we should make LinearProgress animation in the InfoNav continuous. So it animates from width 0-100% smoothly and we control it using the duration.

- The initial load looks a little jumpy. There's a flash of what looks like a grey square and you can see the WebGL spin for a split second while the UI opacity fades in. You can either remove the initial fade animation ( keeping it on the text and large images ) or add a solid white background cover to hide those on initial load.

File: public/locales/cn/info.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "step-1": "Tari Universe makes it easy to mine Tari tokens on a Mac or PC. The future financial system is accessible to all {{emoji}} {{emoji1}}",
+    "step-2": "Tari is a groundbreaking layer 1 blockchain protocol. It is lightning-fast and will power magical user experiences. Tari tokens are the money of the Tari network.",
+    "step-3": "Mining Tari is a race. The first computer on the network to solve the puzzle will be rewarded in Tari tokens (XTM) and gems. This process secures the Tari network.",
+    "step-4": "Tari is currently in testnet mode. This is the dress rehearsal before the Tari mainnet launch. Tokens mined now (tXTM) have no value. However, gems do!",
+    "step-5": "Nothing! Once you're synced, you will start mining and earning gems automatically. Earn more gems by mining tXTM and sharing your invite link with friends. The more gems you earn, the larger your future airdrop!",
+    "step-6": "Tari is pro-human, pro-freedom, and open-source. It is powered and secured by everyone running Tari Universe. Tari exists because of you. {{emoji}}"
+  },
+  "footer": {
+    "heading": "i am a footer"
+  },
```
Comment (brianp) [inline public/locales/cn/info.json:12]: oh are you?

Should this be here?

File: public/locales/de/info.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "content": {
+    "step-1": "Tari Universe makes it easy to mine Tari tokens on a Mac or PC. The future financial system is accessible to all {{emoji}} {{emoji1}}",
+    "step-2": "Tari is a groundbreaking layer 1 blockchain protocol. It is lightning-fast and will power magical user experiences. Tari tokens are the money of the Tari network.",
+    "step-3": "Mining Tari is a race. The first computer on the network to solve the puzzle will be rewarded in Tari tokens (XTM) and gems. This process secures the Tari network.",
+    "step-4": "Tari is currently in testnet mode. This is the dress rehearsal before the Tari mainnet launch. Tokens mined now (tXTM) have no value. However, gems do!",
+    "step-5": "Nothing! Once you're synced, you will start mining and earning gems automatically. Earn more gems by mining tXTM and sharing your invite link with friends. The more gems you earn, the larger your future airdrop!",
+    "step-6": "Tari is pro-human, pro-freedom, and open-source. It is powered and secured by everyone running Tari Universe. Tari exists because of you. {{emoji}}"
+  },
+  "footer": {
+    "heading": "i am a footer"
```
Comment (brianp) [inline public/locales/de/info.json:11]: I wont flag all of them, but you get the point.

**Comment by Anziskavds** [comment]
Tested successfully on MacBook Air 13 inch M1 2020 on Testnet v0.6.4.

**Comment by Tas4tari** [comment]
Tested successfully on Windows 11, app version 0.6.4

**Pull request merged**